tools/flang1/flang1exe/func.c

```diff
@@ -1975,8 +1975,8 @@ rewrite_func_ast(int func_ast, int func_args, int lhs)
        list. Those two integer kind for the exitstat and cmdstat argument
        respectively.
      */
-    ARGT_ARG(newargt, nargs - 2) = mk_cval(size_of(stb.user.dt_int), DT_INT4);    
-    ARGT_ARG(newargt, nargs - 1) = mk_cval(size_of(stb.user.dt_int), DT_INT4);    
```

Comment:
Please avoid formatting code that is not relevant to your bug-fix.

Reply:
The current branch still makes formatting changes in irrelevant parts of the file.